api/src/test/java/org/openmrs/validator/CohortValidatorTest.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.validator;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Cohort;
+import org.openmrs.CohortMembership;
+import org.openmrs.Patient;
+import org.openmrs.test.BaseContextSensitiveTest;
+import org.openmrs.test.Verifies;
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+
+/**
+ * Contains test methods for the {@link CohortValidator}
+ */
+public class CohortValidatorTest extends BaseContextSensitiveTest {
+
+    /**
+     * @see CohortValidator#validate(Object, Errors)
+     */
+    @Test
+    @Verifies(value = "should fail if the patient is voided", method = "validate(Object, Errors)")
+    public void validate_shouldFailIfPatientIsVoided() throws Exception {
+        Cohort cohort = new Cohort(2);
+        Patient patient = new Patient(7);
+        patient.setVoided(true);
+        cohort.addMembership(new CohortMembership(patient));
+        Errors errors = new BindException(cohort, "cohort");
+        new CohortValidator().validate(cohort, errors);
+        Assert.assertEquals(true, errors.hasFieldErrors("members"));
```

Comment:
There is an error message associated to each rejection, you need to check for the expected error message since you can potentially reject a field for different reasons which would mean different error messages.